src/main/java/com/google/sps/servlets/EventsServlet.java

```diff
@@ -106,6 +132,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         Date startTime = (Date) entity.getProperty(START_TIME);
         Date endTime = (Date) entity.getProperty(END_TIME);
         String category = (String) entity.getProperty(CATEGORY);
+        String userId = (String) entity.getProperty(EVENT_CREATOR);
+        long eventId = (long) entity.getProperty(EVENT_ID);
```

Comment:
Refactor the eventEntity into an sps/data/Events.java class, similar to your Users class.

Reply:
Made an EventWrapper class instead of an Event class since I use the Event class from the Google Calendar API -- it's a little messy so let me know if there's a better way of going about it.